core/src/test/resources/reference.conf

```diff
@@ -70,6 +70,11 @@ entityUpsert {
   maxContentSizeBytes = 67108864
 }
 
+entityStatisticsCache {
+  workspacesPerSweep = 5
```

Comment:
How did you arrive at 5?  that seems like a small number

Reply:
It felt like a safe number to start with. My main reason for keeping it low is in case multiple large workspace got picked up in the same sweep. Overriding in rawls.conf is quick enough so we could easily bump it up over time. I haven't really tested this on a database with production-level numbers, so I'm erring on the side of caution.

Although given one of David An's comments in the monitor, I may be re-thinking how the sweeping works in general.